docs/ovn-k8s-overlay.1

```diff
@@ -0,0 +1,123 @@
+.TH OVN-K8S-OVERLAY "1" "Jan 2018" "ovn-kubernetes" "OVN-KUBERNETES User Commands"
+.SH NAME
+ovn-k8s-overlay \- initialize master, minion, gateway
```

Comment:
now it just works for gateway. The master and minion initialization does not happen in ovn-k8s-overlay. Stale 'help'. Got to fix that too.

Reply:
Ah yeah.  Maybe phil was using out-of-date git?  I get:

```
[dcbw@localhost go-controller (master)]$ _output/go/bin/ovn-k8s-overlay --help
NAME:
   ovn-k8s-overlay - run ovn-k8s-overlay to init master, minion, gateway

USAGE:
   ovn-k8s-overlay [global options] command [command options] [arguments...]

VERSION:
   0.0.1

COMMANDS:
     gateway-init  Initialize k8s gateway node
     help, h       Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --help, -h     show help
   --version, -v  print the version
```

Reply:
fixed